ci_build.sh

```diff
@@ -42,10 +42,10 @@ make
 
 # Install the FAT mount helper
 sudo cp test/dosemu_fat_mount.sh /bin/.
-sudo chown root.root /bin/dosemu_fat_mount.sh
+sudo chown root:root /bin/dosemu_fat_mount.sh
 sudo chmod 755 /bin/dosemu_fat_mount.sh
 
 # Install the TAP helper
 sudo cp test/dosemu_tap_interface.sh /bin/.
-sudo chown root.root /bin/dosemu_tap_interface.sh
+sudo chown root:root /bin/dosemu_tap_interface.sh
```

Comment:
Why do we need the chowns at all?

Reply:
Probably don't in the CI environment actually except to follow the install instructions inside the helper. If you were to follow those instructions on your own system, you'll want to make them root owned and owner only writeable as it asks that sudo be configured to run then as root without a passwd.